gubernator/templates/build.html

```diff
@@ -34,6 +34,8 @@
 				% set pod_name = text|parse_pod_name
 				% if pod_name
 					<p>Find <tt>{{pod_name}}</tt> mentions in <a id="{{pod_name|slugify}}" href="/build{{build_dir}}/nodelog?pod={{pod_name}}&junit={{filename|basename}}&wrap=on">log files</a>
+				% else
+					<p>Filter through <a href="/build{{build_dir}}/nodelog?junit={{filename|basename}}&wrap=on">log files</a>
```

Comment:
For example here: https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/kubernetes-e2e-gce/20538/
I was thinking that they could just take the namespace and manually enter that into the text box to start filtering with it.